www/assets/gratipay.js.spt

```diff
@@ -1,17 +1,11 @@
 import os
+from gratipay.utils import http_caching
 js_root = os.path.join(website.project_root, 'js')
 [---]
 catted = []
 for root, dirs, files in os.walk(js_root):
     for filename in files:
-        if filename.endswith('.js'):
-            base = root[len(js_root)+0:]
-            catted.append('/' + ('*'*70) + '/\n')
-            catted.append('/*' + (base + '/' + filename).center(68) + '*/\n')
-            catted.append('/' + ('*'*70) + '/' + '\n\n')
-            content = open(os.path.join(root, filename)).read()
-            content = content.decode('ascii')
-            catted.append(content + '\n')
+        catted.append(http_caching.concat_files(files, root, '.js') + '\n')
```

Comment:
This loop (`for filename in files`) shouldn't be needed, no? Might end up writing the same content more than once, maybe.